CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 
 <!--lint disable no-duplicate-headings-->
 
+# 0.2.1
+
+_2020-11-10_
+
+- Removed quotes from "~/.dir_colors" in **README.md** to fix symlink creation issue
+- Change heading levels in **README.md** to conform to Markdown standards (no skipped levels)
+
```

Comment:
It's nice to see you care about change documentations, but the release notes will be generated automatically when preparing a new release version.

```suggestion
```

- [ ] Please apply the suggestion or remove the entry again manually.